### PR TITLE
FOUR-8257

### PR DESCRIPTION
**Pull request by luNunezProcessmaker**
## Issue & Reproduction Steps
N/A
## Solution
N/A
## How to Test
Describe how to test that this solution works.

## Related Tickets & Packages
https://processmaker.atlassian.net/browse/FOUR-8257

## Code Review Checklist
- [ ] I have pulled this code locally and tested it on my instance, along with any associated packages.
- [ ] This code adheres to [ProcessMaker Coding Guidelines](https://github.com/ProcessMaker/processmaker/wiki/Coding-Guidelines).
- [ ] This code includes a unit test or an E2E test that tests its functionality, or is covered by an existing test.
- [ ] This solution fixes the bug reported in the original ticket.
- [ ] This solution does not alter the expected output of a component in a way that would break existing Processes.
- [ ] This solution does not implement any breaking changes that would invalidate documentation or cause existing Processes to fail.
- [ ] This solution has been tested with enterprise packages that rely on its functionality and does not introduce bugs in those packages.
- [ ] This code does not duplicate functionality that already exists in the framework or in ProcessMaker.
- [ ] This ticket conforms to the PRD associated with this part of ProcessMaker.

ci:deploy

### CODE REVIEW

**Comment by pmkerberos** [comment]
QA server was successfully deployed https://ci-a6d8d164da.eng.processmaker.net

File: storage/api-docs/api-docs.json
Comment (pmPaulis) [inline storage/api-docs/api-docs.json:1]: Please remove those changes

**Review by pmPaulis — CHANGES_REQUESTED** [review]

**Comment by pmkerberos** [comment]
QA server was successfully deployed https://ci-a6d8d164da.eng.processmaker.net

**Comment by pmkerberos** [comment]
QA server was successfully deployed https://ci-a6d8d164da.eng.processmaker.net

**Review by pmPaulis — CHANGES_REQUESTED** [review]

**Comment by pmkerberos** [comment]
QA server was successfully deployed https://ci-a6d8d164da.eng.processmaker.net

**Comment by pmkerberos** [comment]
QA server was successfully deployed https://ci-a6d8d164da.eng.processmaker.net

**Comment by pmkerberos** [comment]
QA server was successfully deployed https://ci-a6d8d164da.eng.processmaker.net

**Review by pmPaulis — APPROVED** [review]

**Comment by pmkerberos** [comment]
QA server was successfully deployed https://ci-a6d8d164da.eng.processmaker.net

**Comment by pmkerberos** [comment]
QA server was successfully deployed https://ci-a6d8d164da.eng.processmaker.net

**Comment by pmkerberos** [comment]
QA server was successfully deployed https://ci-a6d8d164da.eng.processmaker.net

**Comment by pmkerberos** [comment]
QA server was successfully deployed https://ci-a6d8d164da.eng.processmaker.net

**Comment by pmkerberos** [comment]
QA server was successfully deployed https://ci-a6d8d164da.eng.processmaker.net

**Pull request merged**